packages/odyssey-storybook/src/components/odyssey-mui/Button/Button.stories.tsx

```diff
@@ -72,72 +181,98 @@ export const ButtonPrimary: StoryObj<ButtonProps> = {
   },
 };
 
-export const ButtonSecondary: StoryObj<ButtonProps> = {
+export const ButtonSecondary: StoryObj<ButtonStoryProps> = {
+  name: "Secondary",
   args: {
     text: "Add crew",
     variant: "secondary",
   },
 };
 
-export const ButtonDanger: StoryObj<ButtonProps> = {
+export const ButtonDanger: StoryObj<ButtonStoryProps> = {
+  name: "Danger",
   args: {
     text: "Add crew",
     variant: "danger",
   },
 };
 
-export const ButtonFloating: StoryObj<ButtonProps> = {
+export const ButtonFloating: StoryObj<ButtonStoryProps> = {
+  name: "Floating",
   args: {
     text: "Add crew",
     variant: "floating",
   },
 };
 
-export const ButtonSmall: StoryObj<ButtonProps> = {
+export const ButtonSmall: StoryObj<ButtonStoryProps> = {
+  name: "Small",
   args: {
     text: "Add crew",
     size: "small",
   },
 };
 
-export const ButtonMedium: StoryObj<ButtonProps> = {
+export const ButtonMedium: StoryObj<ButtonStoryProps> = {
+  name: "Medium",
   args: {
     text: "Add crew",
     size: "medium",
   },
 };
 
-export const ButtonLarge: StoryObj<ButtonProps> = {
+export const ButtonLarge: StoryObj<ButtonStoryProps> = {
+  name: "Large",
   args: {
     text: "Add crew",
     size: "large",
   },
 };
 
-export const ButtonFullWidth: StoryObj<ButtonProps> = {
+export const ButtonFullWidth: StoryObj<ButtonStoryProps> = {
+  name: "Full-width",
   args: {
     text: "Add crew",
     isFullWidth: true,
   },
 };
 
-export const ButtonPrimaryDisabled: StoryObj<ButtonProps> = {
+export const ButtonPrimaryDisabled: StoryObj<ButtonStoryProps> = {
+  name: "Disabled",
+  parameters: {
+    docs: {
+      description: {
+        story:
+          "Disabled buttons should be paired with a Tooltip to provide additional context. A tooltip can be added by setting the `tooltipText` prop on the button to a string.",
+      },
+    },
+  },
   args: {
     text: "Add crew",
     isDisabled: true,
   },
 };
 
-export const ButtonWithIcon: StoryObj<ButtonProps> = {
+export const ButtonWithIcon: StoryObj<ButtonStoryProps> = {
+  name: "Icon",
   args: {
     text: "Add crew",
-    startIcon: <AddIcon />,
+    startIcon: "add",
   },
 };
```

Comment:
My issue with this change is that while you'll be able to choose any icon, it doesn't resemble the way we expect people to use them with `Button`.